tests/aws/test_client.py

```
@@ -29,187 +29,196 @@ def test_instance_aws_client_empty_mpa_account_id(config, aws_client_factory):
+    )
+
+    result = get_paged_response(method_mock, "ResponsibilityTransfers", {})
 
+    assert result == [{"id": "transfer1"}]
+    assert method_mock.call_args_list == [mocker.call()]
 
-def test_invite_organization_to_transfer_billing(
-    config, aws_client_factory, mock_get_paged_response
-):
+
+def test_invite_org_to_transfer_billing(config, aws_client_factory, mock_get_paged_response):
```

Ruff (0.14.13)

Unused function argument: `mock_get_paged_response`

(ARG001)